src/keys/navigation-key.js

```diff
@@ -0,0 +1,58 @@
+import {fireEvent} from '@testing-library/dom'
```

Comment:
What are your opinions on using this kind of code organization? I noticed there is 3 files for typing:

1. utils
1. type
1. type-modifiers

I think navigation keys will end up adding a chunk (but not massive) amount of code. I opted for a new file but I am willing to move things around.

Reply:
I probably wouldn't have bothered making the new file myself, but I'm ok with it.